src/app/round-template.ts

```diff
@@ -0,0 +1,108 @@
+import { FlowerSpecies, allFlowerSpecies } from './flowers';
+import { TimePeriod } from './time-period';
+import { BeeSpecies, allBeeSpecies } from './bees';
+
+export interface RoundTemplate {
+  name: string;
+  flowerSpecies: FlowerSpecies[];
+  startTime: TimePeriod;
+  endTime: TimePeriod;
+  tickSpeed: number;
+  bees?: {
+    species: BeeSpecies;
+    weight: number;
+  }[];
```

Comment:
Oh, I see y'all decided to make the weights non-optional?

Cool cool :slightly_smiling_face: